wordpress-mobile/WordPress-iOS
# Fix scroll issues on tapping images

Before this PR if you touch/tap images around there is scrolling on the textview. This cause some big confusion when moving around in the post.

To test:
 - Open a post with an image
 - Tap on image
 - Check if there is no scroll around when you tap on the image

Needs review: @jleandroperez 

Note: There is some edge cases when the cursor is on text and then two images are together without space that cause the scroll to happen, but will need to be change by a change on the Aztec pod.
